Windows install.ps1 fails: missing ICU header unicode/uregex.h

## Summary
Running install.ps1 (or go install) fails on Windows due to missing ICU headers from go-icu-regex.

## Environment
- OS: Windows
- Go: go1.25.6
- Install method: install.ps1 ("irm https://raw.githubusercontent.com/steveyegge/beads/main/install.ps1 | iex")

## Error
```
# github.com/dolthub/go-icu-regex/internal/icu
.../go-icu-regex@v0.0.0-20250916051405-78a38d478790/internal/icu/icu.go:8:11: fatal error: unicode/uregex.h: No such file or directory
    8 | // #include "unicode/uregex.h"
      |           ^~~~~~~~~~~~~~~~~~
compilation terminated.
```

## Notes
This appears related to other ICU header issues:
- #1444 (Fedora: unicode/uregex.h missing)
- #1433 (macOS ICU headers/flags)
- #1409 (document build deps)

Likely needs Windows-specific ICU dependency or install docs update.

--- a/docs/INSTALLING.md
+++ b/docs/INSTALLING.md
@@ -148,6 +148,8 @@ Beads now ships with native Windows support—no MSYS or MinGW required.
 irm https://raw.githubusercontent.com/steveyegge/beads/main/install.ps1 | iex
 ```
 
+The script installs a prebuilt Windows release if available. Go is only required for `go install` or building from source.
+
 **Via go install**:
 ```pwsh
 go install github.com/steveyegge/beads/cmd/bd@latest
@@ -160,6 +162,8 @@ cd beads
 go build -o bd.exe ./cmd/bd
 Move-Item bd.exe $env:USERPROFILE\AppData\Local\Microsoft\WindowsApps\
 ```
+
+If you see `unicode/uregex.h` missing while building, use the PowerShell install script instead (it downloads a prebuilt binary).
 
 **Verify installation**:
 ```pwsh

--- a/install.ps1
+++ b/install.ps1
@@ -98,6 +98,92 @@ function Install-WithGo {
     return $true
 }
 
+function Get-WindowsArch {
+    try {
+        $arch = [System.Runtime.InteropServices.RuntimeInformation]::ProcessArchitecture
+    } catch {
+        $arch = $env:PROCESSOR_ARCHITECTURE
+    }
+
+    switch ($arch) {
+        "X64" { return "amd64" }
+        "AMD64" { return "amd64" }
+        "Arm64" { return "arm64" }
+        "ARM64" { return "arm64" }
+        default { return $null }
+    }
+}
+
+function Install-FromRelease {
+    Write-Info "Installing bd from GitHub releases..."
+
+    $arch = Get-WindowsArch
+    if (-not $arch) {
+        Write-WarningMsg "Unsupported Windows architecture. Falling back to source install."
+        return $false
+    }
+
+    $apiUrl = "https://api.github.com/repos/steveyegge/beads/releases/latest"
+    try {
+        $release = Invoke-RestMethod -Uri $apiUrl -Headers @{ "User-Agent" = "beads-install" }
+    } catch {
+        Write-WarningMsg "Failed to fetch latest release: $_"
+        return $false
+    }
+
+    $tag = $release.tag_name
+    if (-not $tag) {
+        Write-WarningMsg "Failed to parse latest release tag."
+        return $false
+    }
+
+    $version = $tag.TrimStart("v")
+    $assetName = "beads_${version}_windows_${arch}.zip"
+    $asset = $release.assets | Where-Object { $_.name -eq $assetName } | Select-Object -First 1
+    if (-not $asset) {
+        Write-WarningMsg "No prebuilt asset found for $assetName. Falling back to source install."
+        return $false
+    }
+
+    $tempRoot = Join-Path ([System.IO.Path]::GetTempPath()) ("beads-install-" + [guid]::NewGuid().ToString("N"))
+    New-Item -ItemType Directory -Path $tempRoot | Out-Null
+    $zipPath = Join-Path $tempRoot $assetName
+
+    try {
+        Write-Info "Downloading $assetName..."
+        Invoke-WebRequest -Uri $asset.browser_download_url -OutFile $zipPath
+
+        Write-Info "Extracting archive..."
+        Expand-Archive -Path $zipPath -DestinationPath $tempRoot -Force
+
+        $bdPath = Join-Path $tempRoot "bd.exe"
+        if (-not (Test-Path $bdPath)) {
+            Write-WarningMsg "bd.exe not found in release archive. Falling back to source install."
+            return $false
+        }
+
+        $installDir = Join-Path $env:LOCALAPPDATA "Programs\bd"
+        New-Item -ItemType Directory -Path $installDir -Force | Out-Null
+
+        Copy-Item -Path $bdPath -Destination (Join-Path $installDir "bd.exe") -Force
+        Write-Success "bd installed to $installDir\bd.exe"
+
+        $Script:LastInstallPath = Join-Path $installDir "bd.exe"
+
+        $pathEntries = [Environment]::GetEnvironmentVariable("PATH", "Process").Split([IO.Path]::PathSeparator) | ForEach-Object { $_.Trim() }
+        if (-not ($pathEntries -contains $installDir)) {
+            Write-WarningMsg "$installDir is not in your PATH. Add it with:`n  setx PATH `"$Env:PATH;$installDir`""
+        }
+    } catch {
+        Write-WarningMsg "Failed to install from releases: $_"
+        return $false
+    } finally {
+        Remove-Item -Path $tempRoot -Recurse -Force -ErrorAction SilentlyContinue
+    }
+
+    return $true
+}
+
 function Install-FromSource {
     Write-Info "Building bd from source..."
 
@@ -277,21 +363,24 @@ function Print-GoInstallInstructions {
 
 $installed = $false
 
-if ($goSupport.Present -and $goSupport.MeetsRequirement) {
-    $installed = Install-WithGo
-    if (-not $installed) {
-        Write-WarningMsg "go install failed; attempting to build from source..."
-        $installed = Install-FromSource
+$installed = Install-FromRelease
+if (-not $installed) {
+    if ($goSupport.Present -and $goSupport.MeetsRequirement) {
+        $installed = Install-WithGo
+        if (-not $installed) {
+            Write-WarningMsg "go install failed; attempting to build from source..."
+            $installed = Install-FromSource
+        }
+    } elseif ($goSupport.Present -and -not $goSupport.MeetsRequirement) {
+        Write-Err "Go 1.24 or newer is required (found: $($goSupport.RawVersion)). Please upgrade Go or use your package manager."
+        Print-GoInstallInstructions
+        exit 1
+    } else {
+        # No Go present - do not attempt to auto-download or auto-install Go.
+        Write-Err "Go is not installed. bd requires Go 1.24+ to build from source."
+        Print-GoInstallInstructions
+        exit 1
     }
-} elseif ($goSupport.Present -and -not $goSupport.MeetsRequirement) {
-    Write-Err "Go 1.24 or newer is required (found: $($goSupport.RawVersion)). Please upgrade Go or use your package manager."
-    Print-GoInstallInstructions
-    exit 1
-} else {
-    # No Go present - do not attempt to auto-download or auto-install Go.
-    Write-Err "Go is not installed. bd requires Go 1.24+ to build from source."
-    Print-GoInstallInstructions
-    exit 1
 }
 
 if ($installed) {

--- a/website/docs/getting-started/installation.md
+++ b/website/docs/getting-started/installation.md
@@ -101,10 +101,14 @@ Beads ships with native Windows support—no MSYS or MinGW required.
 irm https://raw.githubusercontent.com/steveyegge/beads/main/install.ps1 | iex
 ```
 
+The script installs a prebuilt Windows release if available. Go is only required for `go install` or building from source.
+
 **Via go install**:
 ```pwsh
 go install github.com/steveyegge/beads/cmd/bd@latest
 ```
+
+If you see `unicode/uregex.h` missing while building, use the PowerShell install script instead.
 
 ## IDE and Editor Integrations
 

